aegis-web/yo/app/styles/style.css

```diff
@@ -85,9 +85,12 @@ a:focus {
 
 .panel {
   background-color: #fafafa;
-  border: none;
+  border-bottom: 1px solid #2e353d;
+  border-bottom-style: double;
+  border-width: medium;
+  border-left: none;
+  border-right: none;
   border-radius: 0;
-  box-shadow: 0 10px 30px rgba(0, 0, 0, 0.25);
 }
```

Comment:
You shouldn't change this class, it's used in many pages

Reply:
ok. I don't know. I followed the instructions: pull from dev, add, commit, push. Commit 1 and 2 are on dev? I haven't modified its parts.

Reply:
I've add an other class for only the login panel

Reply:
yes those commits are already merged in dev, 17 days ago. Your dev branch also has them. Did you overwrite style.css with an older file you had prepared?

Reply:
no, I didn't. I take his file and I modify it

Reply:
is it ok now?